benchmarks/multimodal/sweep/runner.py

```
@@ -0,0 +1,117 @@
+        model=model,
+        port=port,
+        concurrency=concurrency,
+        request_count=request_count,
+        warmup_count=warmup_count,
+        input_file=input_file,
+        osl=osl,
+        artifact_dir=artifact_dir,
+    )
+
+    print(f"  aiperf concurrency={concurrency} -> {artifact_dir}", flush=True)
+    proc = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)